src/src/MetricsWrapper.js

```diff
@@ -132,6 +132,36 @@ export class MetricsWrapper {
     return counts;
   }
 
+  pprintAwsResourceCountsForCapability(capabilityId) {
```

Comment:
pprint?

Reply:
Yeah. This is some debugging I didn't clean up.
I used it to pretty-print resource counts in the console to check various things.

I'll remove it